.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'feature-request, needs-triage'
```

Comment:
added some label assignment. since template is now "feature-request" matched the label.